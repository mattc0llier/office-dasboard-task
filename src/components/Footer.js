import React from 'react'
import '../../styles/components/footer.scss';

const Footer = () => {
    return(
        <footer>
            <p>GoSquared office dashboard</p>
        </footer>
    )
}

export default Footer