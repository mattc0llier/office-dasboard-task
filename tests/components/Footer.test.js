import React from 'react'
import Footer from '../../src/components/Footer'
import renderer from 'react-test-renderer'

describe('Footer', () => {
    test('matches the footer snapshot', () => {
        const tree = renderer.create(<Footer />).toJSON();
        expect(tree).toMatchSnapshot();
    })
})