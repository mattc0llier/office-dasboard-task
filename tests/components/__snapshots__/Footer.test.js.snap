// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches the footer snapshot 1`] = `
<p>
  GoSquared Office Dashboard
</p>
`;
